cmd/process-agent/README.md

```diff
@@ -19,21 +19,19 @@ git clone git@github.com:DataDog/datadog-agent.git
 cd datadog-agent
 ```
 
-Pull down the latest dependencies via `dep`:
+Note that you must be in `$GOPATH/src/github.com/DataDog/datadog-agent`, NOT `~/dd/datadog-agent`.
```

Comment:
Is this worth mentioning? Previous section clearly stated that the repo is cloned in `$GOPATH/src/github.com/DataDog`

Reply:
It might not be necessary - I added it because I ran into that issue. I had the repo already cloned, and I didn't realize the symlink would cause issues.

Reply:
Open to whatever you think!

Reply:
I don't have a strong opinion either, let's keep it for now.